.github/test/selenium_test_editor.py

```diff
@@ -32,24 +32,74 @@
 driver = webdriver.Chrome(service=service, options=chrome_options)
 wait = WebDriverWait(driver, 20)
 
+username = os.getenv("RULE_EDITOR_USERNAME")
+password = os.getenv("RULE_EDITOR_PASSWORD")
+
 try:
     print("Opening Rule Editor site...")
     driver.get(RULE_EDITOR_URL)
 
+    time.sleep(10)  # wait for the page to load
```

Comment:
Why are theses `sleeps` necessary? Are there no better alternatives through the use of `waits`?